libs/entitlement/bridge/build.gradle

```diff
@@ -16,10 +19,14 @@ configurations {
   }
 }
 
+tasks.named('jar').configure {
```

Comment:
What are you trying to achieve here? This jar is already available by default from the project. If you want to access main23 specific jar you can request `java23` from the consumer instead of introducing a new configuration

Reply:
The point of this is to jam the main23 class files into the main jar. We are reusing the mrjar plugin to setup our java versioned source sets, but then dont' care about the META-INF/versions because patching java.base can't read the MRJAR structure. Instead we carefully load the versioned class we care about, all from the same location in the jar.

Reply:
I plan to followup with a change to the mrjar plugin (or perhaps an alternative plugin) that switches the behavior between putting the resulting files in META-INF/versions vs the main jar root.